### PR TITLE
Add @wesleyc94

### PR DESCRIPTION
**Pull request by wesleyc94**

### CODE REVIEW

File: site/class-2-debug/wesleyc94/index.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  font-family: Roboto, -apple-system, BlinkMacSystemFont, Helvetica Neue, Segoe UI, Oxygen, Ubuntu, Cantarell, Open Sans, sans-serif;
+  max-width: 100%;
+}
+
+body {
+  margin: 0;
+}
+
+svg {
+  width: 100w;
```
Comment (wooorm) [inline site/class-2-debug/wesleyc94/index.css]: w -> vw

File: site/class-2-debug/wesleyc94/index.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  height: 100h;
```
Comment (wooorm) [inline site/class-2-debug/wesleyc94/index.css]: h -> vh

File: site/class-2-debug/wesleyc94/index.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.axis path {
+  fill: no;
```
Comment (wooorm) [inline site/class-2-debug/wesleyc94/index.css]: no -> none

File: site/class-2-debug/wesleyc94/index.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<!--Based on https://bl.ocks.org/mbostock/3887118 by Mike Bostock-->
+<meta charset=utf8>
+<!--Prevent favicon error from showing up-->
+<link rel="shortcut icon" type=image/x-icon href=data:image/x-icon;,>
+<title>@wesleyc94</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<link rel=stylesheet href=../index.css>
```
Comment (wooorm) [inline site/class-2-debug/wesleyc94/index.html]: ../index.css -> index.css

File: site/class-2-debug/wesleyc94/index.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+<svg></svg>
+<script src="https://d3js.org/d3.v4.js"></script>
+<script src=../index.js></script>
```
Comment (wooorm) [inline site/class-2-debug/wesleyc94/index.html]: ../index.js -> index.js

**Comment by wesleyc94** [comment]
Thanks for the feedback! I've updated my files.

**Pull request merged**